src/frontend/src/optimizer/plan_node/logical_insert.rs

```diff
@@ -41,18 +42,21 @@ pub struct LogicalInsert {
     table_version_id: TableVersionId,
     input: PlanRef,
     column_indices: Vec<usize>, // columns in which to insert
+    default_columns: Option<Vec<(usize, ExprImpl)>>, // columns to be set to default
```

Comment:
Is is possible to represent `None` with an empty vec and omit this `Option`?

Reply:
yes... is it a better style?